src/main/scala/com/databricks/spark/csv/package.scala

```diff
@@ -58,6 +81,7 @@ package object csv {
                       compressionCodec: Class[_ <: CompressionCodec] = null): Unit = {
       // TODO(hossein): For nested types, we may want to perform special work
       val delimiter = parameters.getOrElse("delimiter", ",")
+      val quote = parameters.get("quote")
```

Comment:
This would throw an exception is user did not specify quote. We should have a default.